.kres.yaml

```diff
@@ -0,0 +1,31 @@
+---
+kind: auto.CI
+spec:
+  compileGHWorkflowsOnly: true
+---
+kind: common.GHWorkflow
+spec:
+  jobs:
+    - name: default
+      buildxOptions:
+        enabled: true
+        crossBuilder: true
+      runners:
+        - self-hosted
+        - pkgs
+      steps:
+        - name: build
+          nonMakeStep: true
+          command: make
+        - name: login-to-registry
+          conditions:
+            - only-on-tag
```

Comment:
do you want to push only on tag? or you should just push on every `main` branch merge?

(this is a copy-paste from kubelet, which is special)

Reply:
Removed. Thanks, I think there is no need for push on tag here.

Reply:
Removed was a wrong move I guess. You should have had a condition to push only on merges to `main`?

Reply:
Yeah, I also forgot to run rekres. Fixed here: https://github.com/siderolabs/playwright-go/pull/7